### PR TITLE
feat(connector): add connector vs. element template section

### PR DESCRIPTION
**Pull request by nikku**
Closes https://github.com/camunda/team-connectors/issues/214

## What is the purpose of the change

Shape difference of connector vs. element template, a topic that did pop up a couple of times already.

## Are there related marketing activities

No.

## When should this change go live?

ASAP.

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/connectors/introduction.md [hunk 1]
```diff
@@ -88,6 +88,12 @@ It depends on your use case.
 - You want to focus on your worker's logic and have no need for using low-level Camunda 8 API? - Write a Connector.
 - You want to provide a standardized modeling experience alongside your runtime behavior? - Write a Connector.
 
+### Connector vs. Element Template
+
+The domain specific UI for modeling a connector is defined through a [connector template](./custom-built-connectors/connector-templates.md), which, in fact, is an [element template](../modeler/desktop-modeler/element-templates/about-templates.md).
```
Comment (tmetzke) [inline docs/components/connectors/introduction.md]:
```suggestion
You define the domain-specific UI for modeling a Connector through a [Connector template](./custom-built-connectors/connector-templates.md). This Connector template is, in fact, an [element template](../modeler/desktop-modeler/element-templates/about-templates.md). 
```

File: docs/components/connectors/introduction.md [hunk 2]
```diff
@@ -88,6 +88,12 @@ It depends on your use case.
...
+
+You can build a connector like system using element templates and plain old job workers, however you may miss out on some benefits (cf. [connector vs. job workers](#connector-vs-job-worker)).
```
Comment (tmetzke) [inline docs/components/connectors/introduction.md:95]:
```suggestion
You can build a Connector-like system using element templates and plain old job workers. However, using a Connector instead brings along some benefits as outlined in [Connector vs. Job Worker](#connector-vs-job-worker)).
```

**Review by tmetzke — CHANGES_REQUESTED** [review]
Nice! I like this. When we are settled for the text, please also add this to `versioned_docs` for `8.0` and `8.1` 🙂

File: docs/components/connectors/introduction.md [hunk 3]
```diff
@@ -88,6 +88,12 @@ It depends on your use case.
 - You want to focus on your worker's logic and have no need for using low-level Camunda 8 API? - Write a Connector.
 - You want to provide a standardized modeling experience alongside your runtime behavior? - Write a Connector.
 
+### Connector vs. Element Template
+
+The domain specific UI for modeling a connector is defined through a [connector template](./custom-built-connectors/connector-templates.md), which, in fact, is an [element template](../modeler/desktop-modeler/element-templates/about-templates.md).
```
Comment (tmetzke) [inline docs/components/connectors/introduction.md:93]:
```suggestion
You define the domain-specific UI for modeling a Connector through a [Connector template](./custom-built-connectors/connector-templates.md). This Connector template is, in fact, an [element template](../modeler/desktop-modeler/element-templates/about-templates.md).
```

**Comment by nikku** [comment]
Will backport once we settled for the text.

File: docs/components/connectors/introduction.md [hunk 4]
```diff
@@ -88,6 +88,12 @@ It depends on your use case.
 - You want to focus on your worker's logic and have no need for using low-level Camunda 8 API? - Write a Connector.
 - You want to provide a standardized modeling experience alongside your runtime behavior? - Write a Connector.
 
+### Connector vs. Element Template
+
+You define the domain-specific UI for modeling a Connector through a [Connector template](./custom-built-connectors/connector-templates.md). This Connector template is, in fact, an [element template](../modeler/desktop-modeler/element-templates/about-templates.md).
+
+You can build a Connector-like system using element templates and plain old job workers. However, using a Connector instead brings along some benefits as outlined in [Connector vs. Job Worker](#connector-vs-job-worker)).
```
Comment (tmetzke) [inline docs/components/connectors/introduction.md:95]:
```suggestion
You can build a Connector-like system using element templates and plain old job workers. However, using a Connector instead brings along some benefits as outlined in [Connector vs. Job Worker](#connector-vs-job-worker).
```

File: versioned_docs/version-8.0/components/connectors/introduction.md [hunk 5]
```diff
@@ -90,6 +90,12 @@ It depends on your use case.
 - You want to focus on your worker's logic and have no need for using low-level Camunda 8 API? - Write a Connector.
 - You want to provide a standardized modeling experience alongside your runtime behavior? - Write a Connector.
 
+### Connector vs. Element Template
+
+You define the domain-specific UI for modeling a Connector through a [Connector template](./custom-built-connectors/connector-templates.md). This Connector template is, in fact, an [element template](../modeler/desktop-modeler/element-templates/about-templates.md).
+
+You can build a Connector-like system using element templates and plain old job workers. However, using a Connector instead brings along some benefits as outlined in [Connector vs. Job Worker](#connector-vs-job-worker)).
```
Comment (tmetzke) [inline versioned_docs/version-8.0/components/connectors/introduction.md:97]:
```suggestion
You can build a Connector-like system using element templates and plain old job workers. However, using a Connector instead brings along some benefits as outlined in [Connector vs. Job Worker](#connector-vs-job-worker).
```

File: versioned_docs/version-8.1/components/connectors/introduction.md [hunk 6]
```diff
@@ -90,6 +90,12 @@ It depends on your use case.
 - You want to focus on your worker's logic and have no need for using low-level Camunda 8 API? - Write a Connector.
 - You want to provide a standardized modeling experience alongside your runtime behavior? - Write a Connector.
 
+### Connector vs. Element Template
+
+You define the domain-specific UI for modeling a Connector through a [Connector template](./custom-built-connectors/connector-templates.md). This Connector template is, in fact, an [element template](../modeler/desktop-modeler/element-templates/about-templates.md).
+
+You can build a Connector-like system using element templates and plain old job workers. However, using a Connector instead brings along some benefits as outlined in [Connector vs. Job Worker](#connector-vs-job-worker)).
```
Comment (tmetzke) [inline versioned_docs/version-8.1/components/connectors/introduction.md:97]:
```suggestion
You can build a Connector-like system using element templates and plain old job workers. However, using a Connector instead brings along some benefits as outlined in [Connector vs. Job Worker](#connector-vs-job-worker).
```

File: docs/components/connectors/introduction.md (hunk 4 above)
Comment (nikku) [reply docs/components/connectors/introduction.md:95]: Woopsie :D

**Review by tmetzke — APPROVED** [review]
Looks really good from my end 👍

**Review by igpetrov — DISMISSED** [review]

**Review by christinaausley — APPROVED** [review]
Lowercased element template and job worker in my latest commit. LGTM 👍

**Pull request merged**